### ISSUE TITLE
Restrict to one instance per form

### ISSUE THREAD

**Issue by binarykitchen** [kjohnson/ninja-forms-videomail#23]
Since we can only have one Videomail-Client instance, we should make sure it only exists as a single field within the Ninja Form.

For this to happen, remove the duplicate button and do not show the videomail field when adding new fields.

**Comment by binarykitchen** [comment]
Probably tricky, eh @kjohnson ?

**Comment by kjohnson** [comment]
Yeah, this is something that we are looking to add to the form builder but isn't ready.

We'd probably be better to operate the frontend JS as a singleton for now.

**Comment by binarykitchen** [comment]
Ok. Is there a nina-forms ticket we could link to here which adds this to the form builder? @kjohnson

**Comment by kjohnson** [comment]
https://github.com/wpninjas/ninja-forms/issues/1585

**Comment by binarykitchen** [comment]
@kjohnson looks like https://github.com/wpninjas/ninja-forms/issues/1585 is closed. so can this be done now? if so, how?
